src/helpers/script_helpers.sh

```diff
@@ -191,6 +191,41 @@ function skippable {
 }
 
 
+# Adds a step to the current command
+function step {
```

Comment:
How about naming this `add_step`. `step` doesn't say enough to me about what the function does